circle.yml

```diff
@@ -18,7 +18,7 @@ dependencies:
   cache_directories:
     - "~/cache"
   override:
-    - sudo apt-get -q update; sudo apt-get install -q -y libgif-dev
+    - sudo apt-get -q update; sudo apt-get install -q -y libgif-dev libsasl2-dev
```

Comment:
The pyldap library is a wrapper around the native openLDAP library, which this installs. Do you want me to add this to the Dockerfile?